src/redux/reducer.js

```diff
@@ -0,0 +1,51 @@
+import { combineReducers } from 'redux'
+
+import { SEARCH_MOVIES, MOVIES_FETCH_SUCCEEDED, DETAILS_FETCH_SUCCEEDED, ERROR } from './actions'
+
+const searchReducerDefaultState = {results: [], remainingResults: 0, searchString: '', isLoading: false}
```

Comment:
The convention is to call `searchReducerDefaultState` --> `INITIAL_STATE`

Reply:
What about defining different initial states for different reducer?